needrestart_info.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+#
+#
+# Description: Expose metrics from needrestart.
+#
+# This script runs needrestart in batch mode. It will never ask for input
+# and will never restart or upgrade anything.
+#
+# Dependencies: python >= 3.5, python3-prometheus-client, needrestart
+#
+# Authors: RomainMou
```

Comment:
```suggestion

"""
Description: Expose metrics from needrestart.

This script runs needrestart in batch mode. It will never ask for input
and will never restart or upgrade anything.

Dependencies: python >= 3.5, python3-prometheus-client, needrestart

Authors: RomainMou
"""
```

This is considered as module docstring by Python interpreters.